scripts/jenkins/ardana/ansible/group_vars/all/versioned.yml

```diff
@@ -42,6 +42,8 @@ versioned_features:
     enabled: "{{ when_cloud9 }}"
   glance-registry:
     enabled: "{{ when_cloud8 }}"
+  manila-share:
```

Comment:
Just to make sure this is indeed your intent: this means that you will not deploy `manila-share` in cloud9 at all, only in cloud8. Is there any particular reason why cloud9 doesn't need `manila-share` ? 

Also please note that there already is a version based condition for the entire manila feature:

```
manila:
    enabled: "{{ when_staging or when_cloud9 }}"		    enabled: "{{ when_staging or when_cloud9 }}"
```

, which means that manila is not enabled at all when testing against cloud8 official media (e.g. when testing maintenance updates). This was added some time ago because we haven't released all manila related maintenance updates. Do you know if that is still the case ?

Reply:
We decide to disable manila-share in the input model in c9(to make it as dummy one) as in this review https://gerrit.suse.provo.cloud/#/c/5684/  The reason being manila-api and manila-share  are collocating on the controller. I think we still need that once we decide to unlock the condition for manila in CI testing. As for 2nd question, I don't have answer for that. BTW, I triggered a stagingcloud8 job(#3954), not sure why manila-deploy is commented out. Isn't it shall bypass that filter?

Reply:
In [your job](https://ci.suse.de/blue/organizations/jenkins/openstack-ardana/detail/openstack-ardana/3954/pipeline/27) , you chose to use an existing input model from the `ardana-input-model` repository, because you used the `model` parameter.
What you want to do is to use the `scenario_name` parameter, same as [the gating job for cloud8](https://ci.suse.de/view/Cloud/view/Cloud8%20Ardana/job/cloud-ardana8-job-std-min-x86_64/505/parameters/).